odoo/modules/migration.py

```diff
@@ -58,6 +58,12 @@ def __init__(self, cr, graph):
         self._get_files()
 
     def _get_files(self):
+        def _get_migrations_path(pkg):
+            migration_path = opj(tools.config['migrations_path'], pkg)
+            if os.path.exists(migration_path):
+                return migration_path
+            return False
```

Comment:
None is a better value
```suggestion
            return None
```